src/components/common/Footer/index.tsx

```diff
@@ -67,6 +67,9 @@ const Footer = (): ReactElement | null => {
             <li>
               <FooterLink href={getHref(AppRoutes.settings.index)}>Preferences</FooterLink>
             </li>
+            <li>
+              <FooterLink href={HELP_CENTER_URL}>Help</FooterLink>
```

Comment:
I would suggest to make it an external link and open in a new tab. Should we also track its usage?